usermods/Battery/battery_defaults.h

```diff
@@ -0,0 +1,67 @@
+// pin defaults
+// for the esp32 it is best to use the ADC1: GPIO32 - GPIO39
+// https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/peripherals/adc.html
+#ifndef USERMOD_BATTERY_MEASUREMENT_PIN
+  #ifdef ARDUINO_ARCH_ESP32
+    #define USERMOD_BATTERY_MEASUREMENT_PIN 32
```

Comment:
I'd suggest you use gpio35 by default, for two reasons:
* gpio 32 is also used as default pin in audioreactive
* on many "lolin32" boards, gpio35 is connected to battery voltage divider circuit already

Reply:
Thanks for the suggestion 👍.
I should probably order 2-3 esp32. I don't have much experience with 32s yet.
The only project I have used them on was a LoRa project.